doc/long-help.txt

```diff
@@ -155,6 +155,17 @@ Options:
           escape sequences unless the syntax highlighting language is plain text. Possible values:
           auto, always, *never*.
 
+      --disable-line-limits
+          Disables all line limits. Short for `--soft-line-limit 0 --hard-line-limit 0`.
+
+      --soft-line-limit <BYTES>
```

Comment:
I'm thinking about the name for this. What name for this setting do other editors use?